File: Examples/Rust/src/examples/file_connection.rs
use super::connection;
use ximu3::connection_config::*;

pub fn run() {
    let config = &ConnectionConfig::FileConnectionConfig(FileConnectionConfig {
        file_path: "C:/x-IMU3 Example File.ximu3".to_owned(),
    }); // replace with actual connection config

    connection::run(config);
}

File: Examples/C/examples/file_connection.c
#include "connection.h"
#include "Ximu3.h"

void file_connection() {
    const XIMU3_FileConnectionConfig config = (XIMU3_FileConnectionConfig){
        .file_path = "C:/x-IMU3 Example File.ximu3",
    }; // replace with actual connection config

    XIMU3_Connection *const connection = XIMU3_connection_new_file(config);

    run(connection);
}
